### ISSUE TITLE
Please Is there any way to do pagination with geofirestore

### ISSUE THREAD

**Issue by jerryheir** [geofirestore/geofirestore-js#149]
I am working on a project and I can't seem to find a way on the documentation on how to paginate data like when using .startAfter when querying without using geofirestore.
My question is, how do I paginate data being fetched using geofirestore

**Comment by JorgeMoralesLopez** [comment]
It's impossible the way you could paginate with Firestore....
https://github.com/geofirestore/geofirestore-js/issues/144

**Comment by jerryheir** [comment]
Wow. Straight up. No workaround. Nothing. . . That's not scalable at all

**Comment by jerryheir** [comment]
what can we do?

**Comment by MichaelSolati** [comment]
Thanks @JorgeMoralesLopez, unfortunately because of the complexity and the fact that geohash queries aren't inherently suited for pagination this isn't something geofirestore will support.

**Comment by JorgeMoralesLopez** [comment]
@jerryheir What I do is increase the limit, I know it is a poorly optimized page but there is no other formula. What I did is create two types of users, one short and one long ... The short one is the one I use with geofirestore and it is the one I show first ... if the one who is using the app wants more details because when it comes in it does a request to the user with more details. Also putting heavy things like images in subcollections helps.
The truth is that it hurt when I knew that I could not do real paging with geofirestore or with firestore using geopoints, I already had the MPV quite advanced and I did not know what other option to take, but thanks to @MichaelSolati and geofirestore I have been able to follow this way
